keyboards/system76/system76_ec.c

```diff
@@ -157,22 +215,31 @@ void raw_hid_receive(uint8_t *data, uint8_t length) {
 #if defined(RGB_MATRIX_CUSTOM_KB)
         case CMD_LED_GET_VALUE:
             if (!bootloader_unlocked) {
-                if (data[2] == CMD_LED_INDEX_ALL) {
-                    data[3] = rgb_matrix_config.hsv.v;
-                    data[4] = RGB_MATRIX_MAXIMUM_BRIGHTNESS;
-                    data[1] = 0;
+                uint8_t index = data[2];
+                for(uint8_t layer = 0; layer < DYNAMIC_KEYMAP_LAYER_COUNT; layer++) {
+                    if (index == (0xF0 | layer)) {
+                        data[3] = layer_rgb[layer].hsv.h;
```

Comment:
This should be `hsv.v`, not `hsv.h`, right?

Reply:
True